### PR TITLE
Rholang: Normalizer: Add support for send.

### PR DESCRIPTION
**Pull request by guardbotmk3**

### CODE REVIEW

File: rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala [hunk 1]
```diff
@@ -165,6 +165,28 @@ object ProcNormalizeMatcher {
...
+        val dataResults = (initAcc /: p.listproc_.asScala.toList.reverse)(
+          (acc, e) => {
+            val procMatchResult = normalizeMatch(e, acc._2)
+            (procMatchResult.par :: acc._1,
+             ProcVisitInputs(Par(), input.env, procMatchResult.knownFree))
+          }
+        )
+        val persistent = p.send_ match {
+          case _: SendSingle => false
+          case _: SendMultiple => true
```
Comment (KentShikama) [inline rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala]: My immediate reaction to seeing this was "oh so you can either send a single value or a tuple - as in x!(5) vs x!(5,5)" but then I realized this meant x!(5) vs x!!(5). Maybe a rename? But eh maybe we shouldn't touch the grammar too frequently.

**Comment by KentShikama** [comment]
@echatav From my read it just delegates normalizing the name it is sending on and then it delegates normalizing the data it is actually sending. And then it just repackages the normalized results.

**Comment by KentShikama** [comment]
Oh and it sets the persistent flag (true/false depending on whether its a x!(5) or x!!(5)) and passes on the update knownFree map.

**Comment by MParlikar** [comment]
I"m not a fan of merging PR's that we know break the build.

**Comment by KentShikama** [comment]
@guardbotmk3 I've submitted a PR on https://github.com/guardbotmk3/rchain/pull/2 but it looks pretty ugly without a rebase. Feel free to just cherry-pick off the last commit that does the actual fixes. I can also just submit that PR to dev if you want.

**Comment by guardbotmk3** [comment]
The merge commit fixes the broken build.

**Pull request merged**